bin/taiko.js

```diff
@@ -62,9 +62,13 @@ function getPossibleModulePaths() {
 function loadPlugin(plugin) {
     try {
         let paths = getPossibleModulePaths();
-        let location = paths.find((p) => { return fs.existsSync(path.join(p, 'taiko-' + plugin)); });
+        let location =  paths.map(p => {
+            if(fs.existsSync(path.join(p, plugin))) {
+                return path.join(p, plugin);
+            }
+        }).filter(function(p){return p;})[0];
```

Comment:
The filter function here is not really adding value. Is it needed?

Reply:
@nehashri Yes, it's needed. There could be a use case where the user has installed taiko-plugin locally or globally then location would look like 

[ '/Users/saikrisv/git/taiko/node_modules/taiko-diagnostics', undefined ]

`filter(function(p){return p;})` returns [ '/Users/saikrisv/git/taiko/node_modules/taiko-diagnostics' ] then [0]

Happy if you suggest a better way of writing this.